drawBot/context/svgContext.py

```diff
@@ -35,7 +35,7 @@ def _intGeneratorFunc():
 # simple file object
 
 
-class SVGFile(object):
+class SVGFile():
```

Comment:
I think I commented this before: please remove the empty `()` in class definitions: this is uncommon style, and black would delete it.